_posts/2023-05-22-como-ser-el-mejor-dev-de-tu-equipo.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: Cómo ser el mejor dev de tu equipo
+subtitle: Sobre como contruir confianza 
+background: "/assets/images/2023-05-22-como-ser-el-mejor-dev-de-tu-equipo/portada.png"
+images_path: "/assets/images/2023-05-22-como-ser-el-mejor-dev-de-tu-equipo"
+author: Javier Vergara
+tags: confianza desarrollo
+---
+
+Bajo un título que huele a clickbait espero poder entregarte un consejo que de verdad te ayude en tu carrera como ingeniero de software
+
+No se si te pasa como a mi, que estoy un poco aburrido de tanto consejo “esotérico” pululando por ahí: rutinas avaladas por neurólogos sobre cuál es la hora óptima de dormir o despertar,
+millonarios aconsejando trabajar 12 horas por 7 días a la semana, monstruosos toolset de herramientas para tomar notas y la manera correcta de organizarlas, u otro framework “agile” de moda. 
+
+Por estas razones, me sorprendió encontrarme hace unas semanas con una conferencia bajo el título “Grow Your Own Tech Leads • Ken Scambler • YOW! 2019” en YT, 
+la cual se aleja totalmente de de los consejos esotéricos y vuelve a algo completamente básico y esencial en los equipos, la confianza.
+
+Este artículo no es más que un modesto resumen de la conferencia que mencioné (link al final), y en honor al tiempo que tomaste en leer esta entrada,
+te dejo de inmediato con el consejo que te va a convertir en un ingeniero de otro planeta, redoble de tambores…
+
+ > ## no lo hagas pésimo ##
+
+Medio en broma, medio en serio, así de simple, en inglés el consejo era “don't be terrible”. Ten un poco de paciencia y te explico con unos ejemplos a qué me refiero.
+
+### 1. Escribir la minuta ###
```

Comment:
Buena @javier-vergara-buk . Como te comenté, creo que no se entiende el mensaje de este párrafo, yo pensaba que estabas literalmente mandando a los devs a tomar notas a todas las reuniones.

Quizás sería bueno decir el mensaje principal en el Título... ¿cuál sería ese mensaje? Yo me quedé con "Si dices que lo vas a hacer, hazlo, y si vas a fallar avisa temprano".

En esa línea quizás hay dos cosas que puedan apoyar esa afirmación:
- ["Scar Tissues Make Relationships Wear Out"](https://gist.github.com/gtallen1187/27a585fcf36d6e657db2): Habla de que las relaciones generalmente no mueren por un evento grande, si no son los pequeños problemas y fallos los que la van matando
- [Si vas a fallar, falla temprano](https://rebelsguidetopm.com/if-you-are-going-to-fail-fail-fast/). No sé si es el mejor link pero es lo que encontré jaja

Dale, si quieres pinponeamos el párrafo por slack o por acá, como prefieras 👍🏼

Reply:
puse una ejemplo que me parece sencillo y confunde menos. Va en linea con el primer post que me enviaste ("pequeños problemas") y me sirve de "autofuna" por demorarme en este PR :disappointed: